src/data/imports/v1-sample-profile.json

```diff
@@ -0,0 +1 @@
"Choppin'": 3, "Fishing": 1, "Alchemy": 43, "Catching": 2, "Trapping": 18, "Construction": 43, "Worship": 48}, "items": {"Inventory Bag A": true, "Inventory Bag B": true, "Inventory Bag C": true, "Bummo Bag": true, "Capitalist Case": true, "Miniature Mining Pouch": true, "Cramped Choppin Pouch": true, "Small Food Pouch": true, "Small Materials Pouch": true, "Small Soul Pouch": true}}, {"name": "Bolta", "class": "Hunter", "level": 94, "constellations": {"A-7": true, "A-9": true, "B-4": true, "B-8": true}, "starSigns": {"19": true, "12": true}, "skills": {"Mining": 10, "Smithing": 38, "Choppin'": 1, "Fishing": 1, "Alchemy": 43, "Catching": 13, "Trapping": 24, "Construction": 43, "Worship": 16}, "items": {"Inventory Bag A": true, "Inventory Bag B": true, "Inventory Bag C": true, "Inventory Bag D": true, "Inventory Bag E": true, "Snakeskinventory Bag": true, "Totally Normal And Not Fake Bag": true, "Bummo Bag": true, "Capitalist Case": true, "Wealthy Wallet": true, "Cramped Mining Pouch": true, "Cramped Choppin Pouch": true, "Small Food Pouch": true, "Small Materials Pouch": true, "Cramped Bug Pouch": true, "Small Critter Pouch": true}}, {"name": "Aurorie", "class": "Maestro", "level": 73, "constellations": {"A-5": true, "A-8": true, "B-3": true}, "starSigns": {"19": true, "26": true}, "skills": {"Mining": 33, "Smithing": 41, "Choppin'": 34, "Fishing": 30, "Alchemy": 39, "Catching": 25, "Trapping": 19, "Construction": 44, "Worship": 21}, "items": {"Inventory Bag A": true, "Inventory Bag B": true, "Inventory Bag C": true, "Inventory Bag E": true, "Inventory Bag F": true, "Inventory Bag G": true, "Inventory Bag H": true, "Snakeskinventory Bag": true, "Totally Normal And Not Fake Bag": true, "Bummo Bag": true, "Capitalist Case": true, "Wealthy Wallet": true, "Small Mining Pouch": true, "Small Choppin Pouch": true, "Small Food Pouch": true, "Small Materials Pouch": true, "Small Fish Pouch": true, "Small Bug Pouch": true, "Small Soul Pouch": true}}, {"name": "Taiphun", "class": "Barbarian", "level": 64, "constellations": {}, "starSigns": {"25": true, "20": true}, "skills": {"Mining": 8, "Smithing": 28, "Choppin'": 3, "Fishing": 38, "Alchemy": 40, "Catching": 13, "Trapping": 13, "Construction": 40, "Worship": 1}, "items": {"Inventory Bag A": true, "Inventory Bag B": true, "Inventory Bag C": true, "Inventory Bag E": true, "Bummo Bag": true, "Capitalist Case": true, "Cramped Mining Pouch": true, "Small Food Pouch": true, "Small Materials Pouch": true, "Average Fish Pouch": true}}, {"name": "Petrichora", "class": "Bowman", "level": 76, "constellations": {}, "starSigns": {"19": true, "26": true}, "skills": {"Mining": 10, "Smithing": 27, "Choppin'": 2, "Fishing": 1, "Alchemy": 38, "Catching": 1, "Trapping": 7, "Construction": 38, "Worship": 1}, "items": {"Inventory Bag A": true, "Inventory Bag B": true, "Inventory Bag C": true, "Bummo Bag": true, "Capitalist Case": true, "Miniature Mining Pouch": true, "Small Food Pouch": true, "Small Materials Pouch": true}}]}
```

Comment:
This should probably be updated until it's 100% correct. I sent you updated json with the star signs fixed (hopefully).